src/storybook/story-test-runner/index.ts

```diff
@@ -16,16 +16,35 @@ export function run(stories: StorybookStory[], opts: TestplaneOpts): void {
     withStoryFileDataStories.forEach(story => createTestplaneTests(story, opts));
 }
 
-function createTestplaneTests(story: StorybookStoryExtended, { autoScreenshots }: TestplaneOpts): void {
+function createTestplaneTests(
+    story: StorybookStoryExtended,
+    { autoScreenshots, autoScreenshotStorybookGlobals }: TestplaneOpts,
+): void {
     nestedDescribe(story, () => {
+        const rawAutoScreenshotGlobalSets = Object.keys(story.autoScreenshotStorybookGlobals).length
+            ? story.autoScreenshotStorybookGlobals
+            : autoScreenshotStorybookGlobals;
```

Comment:
Picking "which autoScreenshotStorybookGlobals to use" between`autoScreenshotStorybookGlobals` from `testplane.config.ts` and `autoScreenshotStorybookGlobals` from storyfile, preferring the storyfile one.

If storyfile globals are defined, using it without merging with `testplane.config.ts` version

Reply:
I thought we might encounter situation, when user does not need to test some component with multiple themes.

On the other hand, if this case will happen, we could bring another PR and make `story.autoScreenshotStorybookGlobals` to be type of either Record, either function, where config defaults are passed as first argument, so user will be able to do it.

So, i will merge them